espnet2/bin/asr_inference.py

```diff
@@ -415,21 +421,93 @@ def __call__(
 
                 # c. Passed the encoder result and the beam search
                 ret = self._decode_single_sample(enc_spk[0])
-                assert check_return_type(ret)
+                assert check_return_type(ret)  # FIXME(jiyang): probably broken
```

Comment:
Raise error to tell user using en_s2t_task or multi_asr with intermediate ctc is not supported